speakers.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: noheader
+---
+
+<div class="content-block theme-3" id="speakers">
+  <div class="landmark-content">
+    <h1>
+      Speakers
+    </h1>
+    <ul class="speakers">
+      {% for item in site.data.schedule %}
+      {% if item.bio %}
+      <li class="speakers__item">
+        <div class="speakers__container">
+          <h3 id="#{{ item.name | slugify }}">{{ item.name }}</h3>
```

Comment:
Fel ID här. 😄 Brädgården ska inte vara en del av ID-attributet. När man länkar till ett ID så ska man ha med `#` i URL:en, fast inte här. 😉 
```suggestion
          <h3 id="{{ item.name | slugify }}">{{ item.name }}</h3>
```